libtelemetry/src/main/java/com/mapbox/android/telemetry/AttachmentListener.java

```diff
@@ -0,0 +1,9 @@
+package com.mapbox.android.telemetry;
+
+import java.util.List;
+
+public interface AttachmentListener {
+  void onAttachmentResponse(String message, int code, List<String> fieldIds);
```

Comment:
Seems that here is a typo and `fieldIds` should be `fileIds` instead

Reply:
Being handled in this PR: https://github.com/mapbox/mapbox-events-android/pull/212